pkgs/by-name/bi/bitmagnet/package.nix

```diff
@@ -1,22 +1,22 @@
 {
   lib,
-  buildGo122Module, # builds, but does not start on 1.23
+  buildGo123Module,
```

Comment:
My understanding was that a package should take `buildGoModule` and override it in `pkgs/top-level/all-packages.nix` if using the non-default version (making it easier to update it in the future).

Reply:
Inspired me to open #394058.

Reply:
Yup, let's move that override to all-packages.